charts/promitor-agent-scraper/templates/deployment.yaml

```diff
@@ -27,9 +27,9 @@ spec:
         type: {{ .Values.service.selectorType }}
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
-      {{- if .Values.secrets.createSecret }}
+        {{- if .Values.secrets.createSecret }}
```

Comment:
This change fixes a linting error of yamburger: `missed comma between flow collection entries`.

Reply:
That's fine for me, thanks!